pkg/lib/machine-add.html

```diff
@@ -1,17 +1,16 @@
 <div class="modal-header">
     <button class="close" data-dismiss="modal"
         aria-hidden="true">&times;</button>
-    <h4 class="modal-title" translate="yes">Add a New Host</h4>
+    <h4 class="modal-title" translate="yes">Add new host</h4>
```

Comment:
This is gonna conflict with https://github.com/cockpit-project/cockpit/pull/14574 which is going to land very soon.

Reply:
I'm happy to wait for that to land first